gui-wm/cage/cage-0.1.2.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="wayland kiosk"
+HOMEPAGE="https://www.hjdskes.nl/projects/cage/"
```

Comment:
curl --head https://www.hjdskes.nl/projects/cage/
curl: (6) Couldn't resolve host 'www.hjdskes.nl'

Reply:
Server seems to be down.